src/services/audius-api-client/AudiusAPIClient.ts

```diff
@@ -712,7 +712,18 @@ class AudiusAPIClient {
       args,
       true
     )
-    if (!trackResponse) return null
+    // Try the old route method, ensuring that the track once found has the same owner handle
+    if (!trackResponse) {
```

Comment:
do we do 6 here b/c we know that all the tracks that would have issues were over 100,000? can we comment on that if that's the case?

is it safer to do split(-)[-1] instead?

Reply:
I could query to make sure but let's change to 5 to be certain. I don't like split(-) because that may not return a number and will always match anything with spaces in the title

Reply:
but it would already be in the case of a 404 so something was already pretty wonky w/ the url? a try/catch/NaN check and going to 404 would suffice there as well

Reply:
I think matching digits is the safer route? What's the benefit of doing multiple checks/requests?

Reply:
i think this is probably fine -- but i think potentially had some unthought of edge cases vs. split('-') being super straightforward (to me). im down to ship this the way it is